src/styles/02-molecules/edition-card/_edition-card.scss

```diff
@@ -34,7 +34,6 @@
     background-color: $gray-warm-light;
     max-width: 150px;
     flex: 0 0 75px;
-    display: flex;
```

Comment:
for storytime, ask me about the headaches this caused

Reply:
Oh yikes, okay 😣I'm so sorry, I thought this was going to be an easy CSS fix for you!!

Reply:
Oh it's definitely not you!  It was a subtle thing that caused a lot of things to go badly, and turns out that removing this line is the thing that fixes it. It was a fun adventure to track down.  The moral of the story here is that in the future we need to QA on physical phones as well.

Reply:
We should probably talk about it, because I'm still seeing a bug that I would have expected this to fix—but it's small, so I'm happy to approve this and follow-up with this later:

<img width="533" alt="Screen Shot 2020-05-12 at 10 27 16 AM" src="https://user-images.githubusercontent.com/1720093/81703891-2ac8b780-943b-11ea-9217-5f6b714f0cdf.png">

What were the issues we were seeing on physical phones? Can you send me the QA tickets?

Reply:
Yeah.  Let's move this to slack.